packages/config/src/devices/CompatConfig.ts

```diff
@@ -638,6 +687,69 @@ invalid endpoint index in compat option commandClasses.add`,
 	public readonly endpoints: ReadonlyMap<number, Partial<CommandClassInfo>>;
 }
 
+export interface CompatInterviewCCValue {
+	property: string | number;
+	propertyKey?: string | number;
+	value: unknown;
+}
+
+export class CompatInterviewCC {
```

Comment:
I think this (and the definition) in config files can be simplified by including the endpoint in `CompatInterviewCCValue`, so `values` is always a flat object. You can still separate it into a map by endpoints though (or alternatively `filter()` at the usage site).

Reply:
Hmm.  I considered this at first but then I thought it might be useful to allow additional flags to be defined at the per-endpoint level.

For example, for some devices it might be useful to perform the interview as usual but then overwrite certain values after the fact. In that case, adding a `skipInterview` flag that acts on the endpoint (not a value) would make sense. Similarly, alternative 3 in one of my comments above imagines a `refreshValues` flag for an endpoint.

I also think it's better to maintain the symmetry in the syntax with `commandClasses.add` and `commandClasses.remove` since they all basically work the same way with respect to modeling each endpoint as a separate object. I don't feel strongly about it though, so let me know what you prefer.

Reply:
fair enough